File: Next Greater Element I.js
var nextGreaterElement = function(nums1, nums2) {
  var res = [];
 
  for(let i of nums1){
       var found = false
      const idx = nums2.indexOf(i)
      for(let j = idx+1 ; j<nums2.length ; j++){
if(nums2[j]>i){
res.push(nums2[j])
found =true;
    break;
}
      }
     if(!found)res.push(-1)
   
  }
 return res;
};

File: 496-next-greater-element-i/next-greater-element-i.py
class Solution(object):
    def nextGreaterElement(self, nums1, nums2):
        res = []
        for i in nums1:
            idx  = nums2.index(i)
            found = False
            for j in range(idx , len(nums2)):
                if nums2[j] > i:
                    res.append(nums2[j])
                    found = True
                    break
            if(not found):
                res.append(-1)
        return res
           

      